src/EnergyPlus/SolarShading.cc

```diff
@@ -10366,6 +10380,23 @@ namespace SolarShading {
         }
     }
 
+    int selectActiveWindowShadingControlIndex(int curSurface)
+    {
+        // For a given surface, determine based on the schedules which index to the window shading control list vector should be active
+        int selected = 0; // presume it is the first shading control - even if it is not active it needs to be some shading control which is then turned off in the WindowShadingManager
+        if (Surface(curSurface).windowShadingControlList.size() > 1) {
+            for (std::size_t listIndex = 0; listIndex < Surface(curSurface).windowShadingControlList.size(); ++listIndex) {
+                int wsc = Surface(curSurface).windowShadingControlList[listIndex];
+                //pick the first WindowShadingControl that has a non-zero schedule value
+                if (ScheduleManager::GetCurrentScheduleValue(WindowShadingControl(wsc).Schedule) != 0.0) {
```

Comment:
Should this test be `if >0.0`? I/O Ref description for WindowShadingControl Schedule Name says "If schedule value > 0 , shading control is active, ..."

Reply:
Good catch. I will change